src/pages/ExamsPage/components/AllowanceListActions.test.jsx

```diff
@@ -59,4 +85,50 @@ describe('AllowanceListActions', () => {
       expect(screen.queryByText('Are you sure you want to delete this allowance?')).not.toBeInTheDocument();
     });
   });
+
+  describe('edit modal', () => {
```

Comment:
Still working on getting these to work, but does this test coverage seem good?

Reply:
test cases lgtm